classes/exporter.php

```diff
@@ -74,7 +75,14 @@ public function export($compcatid = 0): \stored_file {
         ];
         $exportxmlfile = $fs->create_file_from_string($filerecord, $this->exportxml($compcatid));
         $exportfiles[constants::FILE_NAME_EXPORT] = $exportxmlfile;
-        $filename = 'tiny_elements_export_' . time() . '.zip';
+
+        $exporttag = 'full';
+        if (!empty($compcatid)) {
+            $compcat = $DB->get_record(constants::TABLES['compcat'], ['id' => $compcatid], 'name', MUST_EXIST);
```

Comment:
For retrieving a single field $DB->get_field(..) is better. See same file,  function exportxml(), line 143:
```
        $categoryname = '';
        if (!empty($compcatid)) {
            $categoryname = $DB->get_field(constants::TABLES['compcat'], 'name', ['id' => $compcatid], MUST_EXIST);
        }
```

Reply:
You are right - changed it!